README.md

```diff
@@ -81,48 +88,49 @@ git clone https://github.com/ResearchKit/ResearchKit.git
 Building
 --------
 
-Build the ResearchKit framework by opening `ResearchKit.xcodeproj` and running the
-`ResearchKit` framework target. Optionally, run the unit tests too.
+Build the *ResearchKit framework* by opening `ResearchKit.xcodeproj` and running the *`ResearchKit`
```

Comment:
The second `ResearchKit` framework is to use to specify a target. Remove the surrounding `*`?